### ISSUE TITLE
There is a Javadoc warning

### ISSUE THREAD

**Issue by javadev** [stleary/JSON-java#885]
```
Warning:  Javadoc Warnings
Warning:  /home/runner/work/JSON-java/JSON-java/src/main/java/org/json/JSONTokener.java:294: warning: no @param for strictMode
Warning:  public String nextString(char quote, boolean strictMode) throws JSONException {
Warning:  ^
Warning: [WARNING] 1 warning
```

**Comment by stleary** [comment]
@javadev Thanks for catching this. I had noticed a warning in local builds but forgot to include a review comment. I have updated #884 to include a fix for the warning.

**Comment by stleary** [comment]
Closing due to problem fixed in #886.
